testFramework/unittests/testsHtmlOutput/AdminCatalogUrlGenerationTest.php

```diff
@@ -101,6 +101,7 @@ public function testUnknownSchemaPage()
         @ini_set('log_errors_max_len', 0);  // unlimited length of message output
         @ini_set('display_errors', 0);      // do not output errors to screen/browser/client
         @ini_set('error_log', TESTCWD . 'log-myDEBUG.txt');
+        @ini_set('error_log', TESTCWD . 'log-myDEBUG.txt');
```

Comment:
Because that's the fix. !!!!!
See https://github.com/facebook/hhvm/issues/3558

One of the suggestions that worked was to duplicate those lines.